templates/_layouts/_class-reference.html

```diff
@@ -0,0 +1,17 @@
+{%- for element, items in data.items() -%}
+<hr>
+<div class="row">
+    <div class="col-2 col-medium-1"><h4 class="p-muted-heading">{{ element }}</h4></div>
+    <div class="col-7 col-medium-5">
+        <div class="row">
+            {%- for classname, description in items.items() -%}
+            <div class="col-3 col-medium-2"><p><code>{{ classname }}</code></p></div>
+            <div class="col-4 col-medium-3">{{ description | markdown }}</div>
+            {%- if not loop.last -%}
+            <div class="col-7 col-medium-5"><hr/></div>
+            {%- endif -%}
+             {%- endfor -%}
```

Comment:
Not a biggie but the indentation is off here

Reply:
```suggestion
            {%- endfor -%}
```